src/test/java/resources/voucherManagementTest.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Voucher Management Api Test
+
+   Scenario: Crate Voucher Api Test
+    Given I can create an VoucherRequestDTO for voucher creation
+    Then I can inject MockServer
+    And I can start mock server
+    When I call the create voucher API with expected status of 202 and stub "/createVoucher"
+    And I can register the stub with "/createVoucher" endpoint for "PUT" request with status of 200
+    When I make the "PUT" request to "/createVoucher" endpoint with expected status of 200
+     Then I should be able to extract response body from callback
+    And I can stop mock server
+
+  Scenario: Activate Voucher Api Test
+    Given I can create an VoucherRequestDTO for voucher activation
+    Then I can inject MockServer
+    And I can start mock server
+    When I call the activate voucher API with expected status of 202 and stub "/activateVoucher"
+    And I can register the stub with "/activateVoucher" endpoint for "PUT" request with status of 200
+    When I make the "PUT" request to "/activateVoucher" endpoint with expected status of 200
+    Then I should be able to verify that the "PUT" method to "/activateVoucher" endpoint received a request with required parameter in activate voucher callback body
+    And I can stop mock server
+
+  Scenario: Redeem Voucher Api Test
+    Given I can create an RedeemVoucherRequestDTO for voucher redemption
+    When I call the redeem voucher API with expected status of 200
+    Then I can assert that redemption was successful by asserting the status in response
+
+    @createAndActivateVoucher @redeemVoucher
```

Comment:
why are we using these annotations here?

Reply:
These test cases need to create and activate new voucher before and redeem the voucher after calling the api this will reduce lines in each scenario